README.md

```diff
@@ -1,54 +1,28 @@
-[![License: LGPL v3](https://img.shields.io/badge/License-LGPL%20v3-blue.svg)](https://www.gnu.org/licenses/lgpl-3.0)
-# BHoM
+[![License: LGPL v3](https://img.shields.io/badge/License-LGPL%20v3-blue.svg)](https://www.gnu.org/licenses/lgpl-3.0) [![Build status](https://ci.appveyor.com/api/projects/status/hvb5pfagqejwvihs/branch/master?svg=true)](https://ci.appveyor.com/api/projects/status/versioning_toolkit/branch/master) [![Build Status](https://dev.azure.com/BHoMBot/BHoM/_apis/build/status/Versioning_Toolkit/Versioning_Toolkit.CheckCore?branchName=master)](https://dev.azure.com/BHoMBot/BHoM/_build/latest?definitionId=128&branchName=master)
```

Comment:
`128` is compliance, `126` is core, Azure badge should be `[![Build Status](https://dev.azure.com/BHoMBot/BHoM/_apis/build/status/Versioning_Toolkit/Versioning_Toolkit.CheckCore?branchName=master)](https://dev.azure.com/BHoMBot/BHoM/_build/latest?definitionId=126&branchName=master)`